subsystems/armSubsystem.py

```diff
@@ -0,0 +1,115 @@
+import wpilib
+import rev
+import constants 
+import commands2
+import phoenix5
+# import playingwithfusion
+
+class ArmSubsystem(commands2.Subsystem):
+    def __init__(self) -> None:
+        super().__init__()
+        
+        self.armmotor1 = rev.CANSparkMax(constants.ELEC.arm_motor1_CAN_ID, rev.CANSparkMax.MotorType.kBrushless)
+        self.armmotor2 = rev.CANSparkMax(constants.ELEC.arm_motor2_CAN_ID, rev.CANSparkMax.MotorType.kBrushless)
+        # self.armmotor1 = rev.CANSparkMax(constants.ELEC.arm_motor1_CAN_ID, rev.CANSparkMax.MotorType.kBrushless)
```

Comment:
When we're ready to merge code into the main branch, it's good to remove stuff that we don't use.  It looks like these lines were used for testing and then commented out.  (And they actually ended up identical with lines 12-13.)  I'd suggest removing lines 14-15.